layouts/base.njk

```diff
@@ -71,6 +70,6 @@
 
 {% block bodyEnd %}
   <script src="{{ '/assets/govuk.js' | url }}"></script>
```

Comment:
Should this have `type="module"` added now?

```suggestion
  <script type="module" src="{{ '/assets/govuk.js' | url }}"></script>
```

Reply:
Of course! Updated.